server/app/views/v1/host_nodes/token.json.jbuilder

```diff
@@ -0,0 +1,2 @@
+json.id @node.node_id
```

Comment:
I would suggest per #2483 that new node APIs use `id: @node.to_path` for consistency with the other APIs, not the deprecated `node_id` used in the existing node APIs.